api-reference/10 UI Widgets/GridBase/1 Configuration/onToolbarPreparing.md

```diff
@@ -99,7 +99,96 @@ The following code shows how you can customize the toolbar using this function.
     <dx-data-grid ...
         (onToolbarPreparing)="onToolbarPreparing($event)">
     </dx-data-grid>
-    
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxDataGrid ...
+            @toolbar-preparing="onToolbarPreparing"
+        />
+    </template>
+
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxDataGrid, DxButton } from 'devextreme-vue/data-grid';
```

Comment:
Are you sure that named import is needed in this case? I thought it should be something like this:

```suggestion
    import { DxDataGrid } from 'devextreme-vue/data-grid';
    import 'devextreme-vue/button';
```

Reply:
Recently, in a similar case in the DataGrid popup editing demo, we faced the issue that DxButton should be imported in React and Vue. So, I added it to code snippets.